gradle/gradle-mvn-mpp-push.gradle

```diff
@@ -35,7 +43,10 @@ task emptyJavadocJar(type: Jar) {
 
 signing {
   required { isReleaseBuild() && gradle.taskGraph.hasTask("uploadArchives") }
-    sign(publishing.publications)
+  sign(publishing.publications)
+  if (hasInMemorySigningKey()) {
```

Comment:
If you're trying to publish locally and don't want to send your secret key, you can also use the standard gpg configuration as long as you don't pass a `PGP_SIGNING_KEY` argument.